cmd/pctl/install.go

```diff
@@ -179,7 +179,7 @@ func install(c *cli.Context) error {
 		gitRepoNamespace = split[0]
 		gitRepoName = split[1]
 	}
-	artifactsMaker := profile.NewProfilesArtifactsMaker(profile.MakerConfig{
+	artifactsMaker := install.NewInstaller(install.InstallerConfig{
```

Comment:
`artifactsMaker` doesn't really fit any longer eh? :)

Reply:
dam thought I caught all of those! good spot :wink: